tests/components/rfxtrx/test_light.py

```diff
@@ -70,62 +74,67 @@ async def test_one_light(hass, rfxtrx):
     )
     await hass.async_block_till_done()
 
-    import RFXtrx as rfxtrxmod
+    state = hass.states.get("light.test")
+    assert state
+    assert state.state == "off"
+    assert state.attributes.get("friendly_name") == "Test"
 
-    hass.data[rfxtrx_core.DATA_RFXOBJECT] = rfxtrxmod.Core(
-        "", transport_protocol=rfxtrxmod.DummyTransport
+    await hass.services.async_call(
+        "light", "turn_on", {"entity_id": "light.test"}, blocking=True
     )
+    state = hass.states.get("light.test")
+    assert state.state == "on"
+    assert state.attributes.get("brightness") == 255
 
-    assert 1 == len(rfxtrx_core.RFX_DEVICES)
-    entity = rfxtrx_core.RFX_DEVICES["213c7f2_16"]
-    entity.hass = hass
-    assert "Test" == entity.name
-    assert "off" == entity.state
-    assert entity.assumed_state
-    assert entity.signal_repetitions == 1
-    assert not entity.should_fire_event
-    assert not entity.should_poll
-
-    assert not entity.is_on
-
-    entity.turn_on()
-    assert entity.is_on
-    assert entity.brightness == 255
-
-    entity.turn_off()
-    assert not entity.is_on
-    assert entity.brightness == 0
-
-    entity.turn_on(brightness=100)
-    assert entity.is_on
-    assert entity.brightness == 100
-
-    entity.turn_on(brightness=10)
-    assert entity.is_on
-    assert entity.brightness == 10
-
-    entity.turn_on(brightness=255)
-    assert entity.is_on
-    assert entity.brightness == 255
-
-    entity.turn_off()
-    assert "Test" == entity.name
-    assert "off" == entity.state
+    await hass.services.async_call(
+        "light", "turn_off", {"entity_id": "light.test"}, blocking=True
+    )
+    state = hass.states.get("light.test")
+    assert state.state == "off"
+    assert state.attributes.get("brightness") is None
 
-    entity.turn_on()
-    assert "on" == entity.state
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {"entity_id": "light.test", "brightness": 100},
+        blocking=True,
+    )
+    state = hass.states.get("light.test")
+    assert state.state == "on"
+    assert state.attributes.get("brightness") == 100
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)